### DISCUSSION TITLE
Factory bootloaders Unlock KIRIN 970 | 960 | 955 | 950 | 935 | 925 | 810 | 710 | 65X | 620(CAM) | 620 | In Theory 980.

### DISCUSSION THREAD

**Discussion by werasik2aa** [kitsuned/PotatoNV#155] [Show and tell]
Based on qualcomm unlock bootloader method.
1. Check fblock locked or not via factory bootloader with [https://werasik2aa.github.io/Huawei-Unlock-Tool](https://github.com/werasik2aa/Huawei-Unlock-Tool/)
3. ( If fblock = LOCKED) If not downgrade the system to emui 10 9 or 8. Recomended the 9/8. Repeat step 1.
If fblock=UNLOCKED flash factory bootloader (fastboot) via factory bootloader (fastboot) . Sound funny? I think no.
4. Then reboot to that flashed bootloader, if it boots and unlocked, next step format userdata and try to boot os.
If os booted and buggy like tray tab not opens, need to patch kernel.img.
6. If it's ok be happy.

*Use Temp unlock from AndroidUtility tool then try hut

(C) [Moongamer ](https://4pda.to/forum/index.php?showuser=7454817) 4pda

**Comment by dirhaam** [comment]
Gonna try it with my huawei matepad 10.4 2020 on saturday. Will give you update later
